cms/envs/common.py

```diff
@@ -1835,16 +1835,33 @@
 
 CREDIT_PROVIDER_SECRET_KEYS = {}
 
-# dir containing all themes
+# .. setting_name: COMPREHENSIVE_THEME_DIRS
+# .. setting_default: []
+# .. setting_description: A list of directories containing themes folders,
+#   each entry should be a full path to the directory containing the theme folder.
 COMPREHENSIVE_THEME_DIRS = []
 
-# Theme directory locale paths
+# .. setting_name: COMPREHENSIVE_THEME_LOCALE_PATHS
+# .. setting_default: []
+# .. setting_description: A list of the paths to themes locale directories e.g.
+#   "COMPREHENSIVE_THEME_LOCALE_PATHS" : ["/edx/src/edx-themes/conf/locale"].
 COMPREHENSIVE_THEME_LOCALE_PATHS = []
 
-# Theme to use when no site or site theme is defined,
-# set to None if you want to use openedx theme
+# .. setting_name: DEFAULT_SITE_THEME
+# .. setting_default: None
+# .. setting_description: Theme to use when no site or site theme is defined, for example
+#   "dark-theme". Set to None if you want to use openedx default theme.
+# .. setting_warning: The theme folder needs to be in 'edx-platform/themes' or define the path
+#   to the theme folder in COMPREHENSIVE_THEME_DIRS.
 DEFAULT_SITE_THEME = None
 
+# .. toggle_name: ENABLE_COMPREHENSIVE_THEMING
+# .. toggle_implementation: DjangoSetting
```

Comment:
@BbrSofiane I can't find a reference on when to set `DjangoSetting` as a `toggle_implementation`, could you please share a reference or remove it otherwise?

Reply:
This is correct. Feel free to improve docs, or hopefully we’ll get to it eventually.

Reply:
@eLRuLL you're right it's not in the document, I believe I saw this in a previous annotations.

@robrap just to clarify, do you mean that 'DjangoSetting' is the correct toggle implementation?

Reply:
yeah this looks correct, I've been seeing this on other PRs as well :+1: